src/wifi/AdafruitIO_WINC1500.h

```diff
@@ -137,7 +137,10 @@ class AdafruitIO_WINC1500 : public AdafruitIO {
         return;
       }
 
-      WiFi.begin(_ssid, _pass);
+      if (_pass && strlen(_pass) > 0)
+        WiFi.begin(_ssid, _pass);
+      else
+        WiFi.begin(_ssid);
```

Comment:
I suggest curly braces here even for single statements. It prevents errors later if more statements are added.